src/Symfony/Component/HttpKernel/HttpCache/HttpCache.php

```diff
@@ -78,6 +78,10 @@ class HttpCache implements HttpKernelInterface, TerminableInterface
      *                            the cache can serve a stale response when an error is encountered (default: 60).
      *                            This setting is overridden by the stale-if-error HTTP Cache-Control extension
      *                            (see RFC 5861).
+     *
+     *   * terminate_on_cache_hit Specifies if the kernel.terminate event should be dispatched even when the cache
```

Comment:
Adding a new option is a new feature :) The only exception would be if it helps fix a security issue.